ui-grid/ui-grid.d.ts

```diff
@@ -490,6 +490,10 @@ declare module uiGrid {
          */
         footerHeight?: number;
         /**
+         * returns or sets grid height in pixels
+         */
+        gridHeight?: number;
```

Comment:
gridHeight is an undocumented read-only property on the grid. As far as I can see from the ui-grid source, setting it to a different value won't change the height of the grid, but will mess up ui-grid's different height calculations.

Whenever I've needed that value myself, I've accessed it using grid['gridHeight'].